platform/linuxbsd/display_server_x11.cpp

```diff
@@ -1850,23 +1850,38 @@ DisplayServerX11::LatinKeyboardVariant DisplayServerX11::get_latin_keyboard_vari
 	_THREAD_SAFE_METHOD_
 
 	XkbDescRec *xkbdesc = XkbAllocKeyboard();
+	if (!xkbdesc)
```

Comment:
You should put braces around single statement blocks.

But here this construct is not needed at all, if you want to provide a description for the error, you can simply replace `ERR_FAIL_COND_V` by `ERR_FAIL_COND_V_MSG`.

Reply:
That being said, I'm not sure either of those added messages are really helpful. `ERR_FAIL_COND` will already print a message about what condition failed, and all this is developer-level information which is useless to end users. And engine developers can check the line where the error was raised to get context.